pulsar-common/src/main/java/org/apache/pulsar/common/util/SimpleTextOutputStream.java

```diff
@@ -43,19 +44,16 @@ public SimpleTextOutputStream write(byte[] a, int offset, int len) {
     }
 
     public SimpleTextOutputStream write(char c) {
-        buffer.writeByte((byte) c);
+        write(String.valueOf(c));
```

Comment:
```suggestion
        buffer.writeChar(c);
```

Reply:
I've tried to use `buffer.writeChar()`, but I doesn't work

Reply:
Oh, the class is used for encoding other types to a UTF-8 buffer.